EAD-Session-8-Assignment/src/main/java/com/metcube/assignment/ServletInitializer.java

```diff
@@ -0,0 +1,13 @@
+package com.metcube.assignment;
```

Comment:
Can you describe in comments when the ServletInitializer will be called in this case.

Reply:
Explained